.github/workflows/ossf-scorecard-reporting.yml

```diff
@@ -27,12 +27,24 @@ jobs:
           scope: tools/ossf_scorecard/scope.json
           database: tools/ossf_scorecard/database.json
           report: tools/ossf_scorecard/report.md
-          auto-commit: true
-          auto-push: true
+          auto-commit: false
+          auto-push: false
           generate-issue: true
           report-tags-enabled: true
           issue-title: "OpenSSF Scorecard Report Updated!"
           github-token: ${{ secrets.GITHUB_TOKEN }}
           max-request-in-parallel: 10
           discovery-enabled: true
           discovery-orgs: 'nodejs'
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@38e0b6e68b4c852a5500a94740f0e535e0d7ba54 # v4.2.4
```

Comment:
I think we might want to use the same action we use on node https://github.com/nodejs/node/blob/92a938b4dd318a8732110da03f1f666db2e96f97/.github/workflows/tools.yml#L285 to open PR

Reply:
Yes, great suggestion @marco-ippolito! I will refactor it (also for #999 and #998)

Reply:
Done in [15ba0fe](https://github.com/nodejs/security-wg/pull/997/commits/15ba0fea0fc9663d3f5b66b18d299375d2635a30)